app/models/concerns/hyrax/ability/work_ability.rb

```diff
@@ -15,9 +15,24 @@ def work_roles
             doc = permissions_doc(id)
             all_work_types_and_files.include?(doc.hydra_model)
           end
-        elsif work_depositor?
+        elsif work_depositor? || admin_set_with_deposit?
           can %i[create], all_work_types_and_files
         end
+
+        # OVERRIDE HYRAX 3.5.0 to return false if no ids are found
+        # @return [Boolean] true if the user has at least one admin set they can deposit into.
+        def admin_set_with_deposit?
+          ids = PermissionTemplateAccess.for_user(ability: self,
+                                                  access: ['deposit', 'manage'])
+                                        .joins(:permission_template)
+                                        .select(:source_id)
+                                        .distinct
+                                        .pluck(:source_id)
+
+          return false if ids.empty?
+
+          Hyrax.custom_queries.find_ids_by_model(model: Hyrax::AdministrativeSet, ids: ids).any?
```

Comment:
What was the error & error message being thrown here? It was only happening outside any tenants, right?

Reply:
ref this hot fix: https://github.com/scientist-softserv/palni-palci/pull/885